src/python/README.md

```diff
@@ -0,0 +1,26 @@
+# How to Create the Python Wrappers
```

Comment:
In think we need to put this file under `doc/python`.

Reply:
Okay, but the instructions will only work from the `src/python` directory. If it is run from somewhere else, the `setup.py` file has the wrong relative path to find `penguinV.i`. Do we intend for users to run `python ..\..\src\python\setup.py build_ext --inplace` from the `doc/python` directory?

If so, I guess a quick workaround is to use relative paths such as `../../src/python/penguinV.i`, since `doc/python` happens to be the same number of levels deep as `src/python`. However, if we intend for `setup.py` to be run from the `doc/python` directory, maybe it would be better to have a separate `setup_doc.py` for that purpose, or to pass another command line argument to `setup.py`. What do you think?

Reply:
The purpose of `doc` directory is obviously to store a documentation. I missed some point that `cuda` and `opencl` have a README files inside their directories with a brief description in them. I think you initial location of the file was correct and I made a wrong suggestion about it as this file contains an information not related to API description. I'll put it back in a separate commit after merging your changes into the project.

The documentation ideally should be generated into `doc/python` directory and we could handle it in a separate issue.